examples/entity_tx.rs

```diff
@@ -31,12 +36,18 @@ fn main() {
     // }
 }
 
-ser_struct! {
-    #[derive(Debug, Clone)]
-    #[allow(non_snake_case)]
-    pub struct Person {
-        crux__db___id: CruxId,
-        first_name: String,
-        last_name: String
-    }
+#[test]
+#[cfg(not(feature = "async"))]
+fn test_entity_tx() {
+    let entity_tx = entity_tx();
+
+    assert!(entity_tx.tx___tx_id > 0);
```

Comment:
id > 0

Reply:
any test works here, because it is mostly to detect if there is no inconsistencies with date, not really going to change to is_empty sorry

Reply:
Also entiity and tx_log don't have a property is_empty 🤔